bedrock/products/templates/products/vpn/more/base-article.html

```diff
@@ -0,0 +1,63 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "products/vpn/includes/macros.html" import vpn_hero, vpn_nav_cta_link, vpn_content_block, with context %}
+
+{% extends "products/vpn/base.html" %}
+
+{% block page_css %}
+  {{ css_bundle('mozilla-vpn-article') }}
+{% endblock %}
+
+{% block site_header %}
+  {% with
+    custom_nav_cta=vpn_nav_cta_link()
+  %}
+    {% include 'includes/protocol/navigation/navigation.html' %}
+  {% endwith %}
+{% endblock %}
+
+{% block sub_navigation %}
+  {% include 'products/vpn/includes/subnav-more.html' %}
+{% endblock %}
+
+{% block content %}
+  {% block hero %}
+    {% call vpn_hero(
+      heading=ftl('vpn-shared-product-name'),
+      subheading=self.page_title(),
+      desc=self.page_desc()
+    ) %}
+    <div class="vpn-hero-cta">
+      <p><a class="mzp-c-button mzp-t-xl" href="{{ url('products.vpn.landing') }}">{{ ftl('vpn-shared-subscribe-link') }}</a></p>
```

Comment:
From a user perspective It feels a little odd that the button reads "Get Mozilla VPN", and when I click it I'm taken to an almost identical looking page with another button that reads "Get Mozilla VPN". I have to click that again and it then scrolls down to the pricing.

Would it be worth considering linking to `{{ url('products.vpn.landing') }}#pricing` to eliminate the second click?